src/generic/datavgen.cpp

```diff
@@ -883,6 +888,7 @@ class wxDataViewMainWindow: public wxWindow
     bool                        m_hasFocus;
     bool                        m_useCellFocus;
     bool                        m_currentColSetByKeyboard;
+    wxHashTable                 *m_heightCache;
```

Comment:
Please use `WX_DECLARE_HASH_MAP` instead of the untyped (`wxObject`-valued) legacy class.